config/manifests/xdescriptors_patch.yaml

```diff
@@ -1,6 +1,9 @@
-- op: replace
-  path: /spec/customresourcedefinitions/owned/2/specDescriptors/3/path
-  value: flightRecorder.name
+- op: add
+  path: /spec/customresourcedefinitions/owned/2/specDescriptors/3
+  value:
+    description: Flight Recorder Name
+    displayName: Name
+    path: flightRecorder.name
```

Comment:
Can we combine these two patches so that we also add the `x-descriptors` array here?